test/test-checkbox.js

```diff
@@ -0,0 +1,122 @@
+var buster = require('buster')
+  , common = require('./common')
+  , assert = buster.referee.assert
+  , refute = buster.referee.refute
+  , testObject
+  ;
+
+buster.testCase('checkbox',
+{
+  setUp: common.setUp,
+
+  'Invokes _triggerEvents with list of events': function(done)
+  {
+    var eventsList = ['touchstart', ['touchend', 'mouseover', 'mousedown', 'focus', 'focusin', 'DOMFocusIn', 'mouseup', '[Function: selectCheckbox]', 'change', 'click'], ['mouseout', 'blur', 'focusout', 'DOMFocusOut']]
+      , target     = common.createTargetElement.call(this)
+      , callback = this.spy()
```

Comment:
Can you clean up formatting? Everything else looks good. Merge at will. Thank you.